DigitalLearningSolutions.Web.Tests/ControllerHelpers/ControllerContextHelper.cs

```diff
@@ -62,6 +62,22 @@ public static class ControllerContextHelper
             return controller;
         }
 
+        public static T WithMockHttpRequestHttpContext<T>(

```

Comment:
Could this be commonised with the two methods above?

Reply:
Removed as no longer needed post refactoring the IsViewPage to a query parameter.